File: WebServer/WebServer/Dto/OrderDto.cs
namespace WebServer.Dto
{
    public class OrderDto
    {
        public List<OrderArticleDto> Articles { get; set; }
        
        public long UserId { get; set; }

        public string Comment { get; set; }

        public string DeliveryAddress { get; set; }

        public double Price { get; set; }
    }
}

File: web2projekat/src/models/OrderDto.js
export default class OrderDto {
    constructor(articles, userId, comment, deliveryAddress, price) {
        this.Articles = articles;
        this.UserId = userId;
        this.Comment = comment;
        this.DeliveryAddress = deliveryAddress;
        this.Price = price;
    }

    addArticle(article) {
        this.articles.push(article);
    }
}